.github/workflows/maven.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Checkout network store sources
+        uses: actions/checkout@v1
+        with:
+          repository: powsybl/powsybl-network-store
+          ref: refs/heads/master
+
+      - name: Build and install network store client with Maven
+        run: mvn --batch-mode -Pclient -DskipTests=true --file ../powsybl-network-store/pom.xml install
+
+      - name: Checkout sources
+        uses: actions/checkout@v1
+
+      - name: Build with Maven
+        run: mvn --batch-mode package
```

Comment:
I already created the project on SonarCloud with key com.powsybl:powsybl-network-conversion-server
 so you can directly add a Sonar step without having any error.